### ISSUE TITLE
nil option object for Field::Extension(s) instead of empty hash

### ISSUE THREAD

**Issue by ngouy** [rmosolgo/graphql-ruby#2182]
Hello

Wanted to know if it's on purpose, and if yes why :

When creating and using a FieldExtension (GraphQL::Schema::FieldExtension) and passing no options, in the `after_resolve` method, `option` object is not an empty hash but is at `nil` instead

So basically if you want an optional option, you have to check the non nillability of option, then check if the option is present then take the value or use a default one, instead of just make something like 
```rb
    def after_resolve(value:, **_)
       value.round(options.fetch(:ndigits) { 0 }) # but have to do value.round(options&.fetch(:ndigits) { 0 } || 0)
    end
```

**Comment by rmosolgo** [comment]
Using an empty hash as default sounds good to me! Do you want to try making that change?

**Comment by ngouy** [comment]
👍 on it
I will try do work this out before next monday

**Comment by ngouy** [comment]
does an edit on the initializer seems the better way to achieve this ?
```
def initialize(field:, options:)
        @field = field
        @options = options || {}
        apply
        freeze
      end
```

**Comment by rmosolgo** [comment]
Any implementation is ok with me, as long as it comes with tests for the new behavior 😉

**Comment by ngouy** [comment]
👌 of course

**Comment by rmosolgo** [comment]
Fixed by #2189
